cpu/esp32/vendor/esp-idf/esp_funcs.c

```diff
@@ -545,3 +545,48 @@ void sc_ack_send_stop(void)
     NOT_SUPPORTED();
 }
 #endif
+
+/*
+ * source: /path/to/esp-idf/components/bootloader_support/src/bootloader_clock.c
+ */
+void bootloader_clock_configure(void)
+{
+    // ROM bootloader may have put a lot of text into UART0 FIFO.
+    // Wait for it to be printed.
+    // This is not needed on power on reset, when ROM bootloader is running at
+    // 40 MHz. But in case of TG WDT reset, CPU may still be running at >80 MHZ,
+    // and will be done with the bootloader much earlier than UART FIFO is empty.
```

Comment:
Just a side comment here, for further reference: `//` comments are not permitted according to our [coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions).

It is not necessary to change it here because it's part of the vendor sources, but for further contributions on the RIOT code you should take it into account.

Reply:
@kYc0o 
> Just a side comment here, for further reference: // comments are not permitted according to our coding conventions.

I saw it too when I reviewed the code. But since it is original code from vendor and placed in the vendor subdirectory, I told @snef that it is not necessary to adopt the code to fulfill RIOT's coding style. It makes no sense to change vendor code.